jdaviz/configs/default/plugins/model_fitting/model_fitting.vue

```diff
@@ -200,6 +200,11 @@
                   </v-btn>
                 </j-tooltip>
               </v-row>
+              <v-row v-if="item.model_type === 'Spline1D'">
+                <v-alert type="info">
+                  To view Spline1D parameters, please open the fitter parameters section below.               </v-alert>
+              </v-row>
+              <template v-else>
               <v-div
                 v-for="param in item.parameters"
```

Comment:
if `item.parameters` is empty for a spline, then we probably don't need to wrap in the `<template v-else>`